core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -80,8 +80,8 @@ import java.util
 import java.util.concurrent.{CompletableFuture, ConcurrentHashMap}
 import java.util.concurrent.atomic.AtomicInteger
 import java.util.{Collections, Optional}
-
```

Comment:
nit: line here -- not sure what the expected format is though 😅

Reply:
reverted this change.